automated install/basic-install.sh

```diff
@@ -1485,7 +1546,7 @@ installConfigs() {
 
     # Install template file if it does not exist
     if [[ ! -r "${PI_HOLE_CONFIG_DIR}/pihole-FTL.conf" ]]; then
-        install -d -m 0755 ${PI_HOLE_CONFIG_DIR}
+        install -d -m 755 ${PI_HOLE_CONFIG_DIR}
```

Comment:
Does this mode change really need to happen?

Reply:
I meant the change from `0755` to `755`. I don't think this line needs to change.

Reply:
I did this because it could have been the reason for the malfunction with `/usr/bin/pihole-FTL`
Also so it is more consistent with the other `install`

Reply:
Shouldn't be the reason. You can make those changes in a separate PR if they are style changes. Please keep this PR to Alpine changes only. 

One PR for one topic will help us review it.

Reply:
Commit reverted